tools/bin/ariaTSsetup.py

```diff
@@ -635,6 +658,7 @@ def main():
     # prepare additional stacks for other layers
     layers += ARIA_STACK_DEFAULTS
     layers.remove('unwrappedPhase')
+    layers = sorted(list(set(layers)))
```

Comment:
I realized that SET (and for that matter any correction layer specified in the commandline) was being duplicated such that the corresponding stack.vrt file was extracted two times.